tests/integ/models.py

```diff
@@ -1,7 +1,93 @@
+from enum import Enum
+
 import random
 import string
 
-from bloop import BaseModel, Column, String
+from bloop import BaseModel, Column, String, UUID, DateTime, Set, Boolean, GlobalSecondaryIndex, LocalSecondaryIndex
+
+
+# ================================================================================
+# Types
+# ================================================================================
+class StringEnum(String):
```

Comment:
:+1: I like that this adds some non-trivial models, everything up to now has been very canned.  It's helpful to see how these features fit into a larger context than the bloop bits.